src/containerapp/azext_containerapp/tests/latest/test_containerapp_workload_profile_commands.py

```diff
@@ -244,6 +244,69 @@ def test_containerapp_create_with_workloadprofile_yaml(self, resource_group):
         clean_up_test_file(containerapp_file_name_01)
         clean_up_test_file(containerapp_file_name_02)
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="eastus")
+    @live_only()  # encounters 'CannotOverwriteExistingCassetteException' only when run from recording (passes when run live)
```

Comment:
May I ask why we need to mark this test as `@live_only()`? I guess this problem may require you to upload the recorded yaml file to solve it

Reply:
incorrectly marked, fixed